specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-03-01-preview/ExportConnections.json

```diff
@@ -0,0 +1,421 @@
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ExportConnection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/2.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ExportConnectionList": {
+      "description": "List all the export connections.",
+      "properties": {
+        "nextLink": {
+          "description": "URL to fetch the next set of export connections.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "value": {
+          "description": "Array of export connections.",
+          "items": {
+            "$ref": "#/definitions/ExportConnection"
+          },
+          "type": "array"
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "type": "object"
+    },
+    "ExportConnection": {
+      "allOf": [
+        {
+          "$ref": "../../../common/2.0/types.json#/definitions/ResourceWithEtag"
+        }
+      ],
+      "description": "Represents a ExportConnection in Azure Security Insights.",
+      "properties": {
+        "properties": {
+          "description": "export connection properties",
+          "$ref": "#/definitions/ExportConnectionProperties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object"
+    },
+    "ExportConnectionProperties": {
+      "description": "Describes export connection properties",
+      "properties": {
+        "connectionId": {
+          "description": "The id (a Guid) of the export connection",
+          "type": "string"
+        },
+        "version": {
+          "$ref": "#/definitions/Version",
+          "description": "The version number associated with the export connection",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "The display name of the export connection",
+          "type": "string"
+        },
+        "description": {
+          "description": "A description of the export connection",
+          "type": "string"
+        },
+        "repoType": {
+          "$ref": "#/definitions/RepoType",
+          "description": "The repository type of the export connection",
+          "type": "string"
+        },
+        "contentTypes": {
+          "description": "Array of export connection content types.",
+          "items": {
+            "$ref": "#/definitions/ContentType"
+          },
+          "type": "array"
+        },
+        "repository": {
+          "description": "Repository metadata.",
+          "$ref": "#/definitions/Repository"
+        },
+        "exportType": {
+          "description": "Export type of the export connection.",
+          "$ref": "#/definitions/ExportType"
+        },
+        "nextScheduledTime": {
```

Comment:
>nextScheduledTime [](http://example.com/codeflow?start=9&length=17)

Add format: date-time, or at minimum describe the format in the description.

Reply:
Added format in the description.